### PR TITLE
edit xiaoli blog

### PR DESCRIPTION
**Pull request by judaschrist**
RT

### CODE REVIEW

File: _posts/2018-08-13-wechaty-xiaoli.md [hunk 1]
```diff
@@ -227,7 +228,7 @@ async function onLogin(user) {
 
 以上代码表示机器人会在周一到周五每天9:00am准时给群里发送一份人工智能日报。
 
-到这里，这个又能查新闻，又能发日报的wechaty机器人就完成啦（完整代码看[这里](https://github.com/judaschrist/wechaty-getting-started/blob/master/examples/advanced/xiaoli-news-bot.js)）。
+到这里，这个又能查新闻，又能发日报的wechaty机器人就完成啦（完整代码看[这里](https://github.com/judaschrist/wechaty-getting-started/tree/master/examples/third-party/xiaoli)）。
```
Comment (huan) [inline _posts/2018-08-13-wechaty-xiaoli.md]: I'd like to suggest you to use a link with a version hash in the url, to prevent the 404 error in the future.

**Review by huan — APPROVED** [review]

File: _posts/2018-08-13-wechaty-xiaoli.md (hunk 1 above)
Comment (lijiarui) [reply _posts/2018-08-13-wechaty-xiaoli.md]: @judaschrist maybe url should be something like this:

https://github.com/Chatie/wechaty-getting-started/tree/1305ada4278e7d19932a2c824e5d7eae5eb41f0f/examples/third-party/xiaoli

**Comment by judaschrist** [comment]
@zixia @lijiarui thanks for the advice! that's a github feature I did not know before. :)

**Comment by lijiarui** [comment]
@judaschrist maybe you should convert from coding.net to GitHub ：）

**Comment by judaschrist** [comment]
@lijiarui i would but not so sure about the service stability within mainland china, 😂😂😂

**Comment by lijiarui** [comment]
developers should use VPN more frequently :)

**Pull request merged**